examples/src/main/java/com/jagrosh/jdautilities/examples/package-info.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 John Grosh (jagrosh).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Common package for all package for all available examples classes.<br>
```

Comment:
What does this even mean...?

Reply:
Should be explained in the comment above already.

Reply:
`... package for all package for all...`
This doesn't sound quite right

Reply:
I didn't even see that :man_facepalming:.